libsplinter/src/service/scabbard/rest_api.rs

```diff
@@ -64,10 +64,12 @@ pub fn make_subscribe_endpoint() -> ServiceEndpoint {
 
             let last_seen_event_id = query.remove("last_seen_event");
 
-            if let Some(ref id) = last_seen_event_id {
-                debug!("Getting all state-delta events since {}", id);
-            } else {
-                debug!("Getting all state-delta events");
+            match last_seen_event_id {
+                Some(ref id) if id.trim().is_empty() => {
```

Comment:
Assuming that the empty string will never be a valid event ID, it's probably fine to just treat it as if the parameter wasn't provided. I think this might be the desired behavior, especially since the response doesn't include any information about why the request was bad, and it would not be immediately obvious to the user.

Reply:
Updated to return a message